src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/DefaultRazorFormattingService.cs

```diff
@@ -85,6 +85,36 @@ public override async Task<TextEdit[]> FormatAsync(Uri uri, RazorCodeDocument co
             return edits;
         }
 
+        public override Task<TextEdit[]> ApplyFormattedEditsAsync(Uri uri, RazorCodeDocument codeDocument, RazorLanguageKind kind, TextEdit[] formattedEdits, FormattingOptions options)
+        {
+            var span = TextSpan.FromBounds(0, codeDocument.Source.Length);
+            var range = span.AsRange(codeDocument.GetSourceText());
```

Comment:
break source text into its own variable. I've been debugging so much in release builds recently having the separate variables is sooooo beneficial and every piece that doesn't end up having it makes me slam my head against the wall